lib/travis/model/job.rb

```diff
@@ -154,10 +155,21 @@ def matches_config?(other)
   end
 
   def log_content=(content)
+    if Travis.config.logs_api.enabled?
+      return RemoteLog.write_content_for_job_id(id, content: content)
+    end
     create_log! unless log
     log.update_attributes!(content: content, aggregated_at: Time.now)
   end
 
+  if Travis::Config.logs_api_enabled?
```

Comment:
Same comment as in the V3 job model, you can call super conditionally and continue to have the former log method behavior.